src/converse-chatview.js

```diff
@@ -67,6 +67,7 @@ converse.plugins.add('converse-chatview', {
                 'emoji': true,
                 'spoiler': true
             },
+            'filter_url_query_params': []
```

Comment:
When adding a new configuration option, please also document it here:
https://github.com/conversejs/converse.js/blob/master/docs/source/configuration.rst

Options are added in alphabetical order.